src/parser/jobs/blu/modules/AlwaysBeCasting.tsx

```diff
@@ -59,26 +61,26 @@ export class AlwaysBeCasting extends CoreAlwaysBeCasting {
 		super.initialise()
 
 		const playerFilter = filter<Event>().source(this.parser.actor.id)
-		const phantomFlurryCastFilter = playerFilter
-			.action(this.data.actions.PHANTOM_FLURRY.id)
+
+		const channelCastFilter = playerFilter
+			.action(oneOf([this.data.actions.PHANTOM_FLURRY.id, this.data.actions.APOKALYPSIS.id]))
```

Comment:
Since were' building this list multiple times, can we make a `private channelCastIds` or something similar?

Reply:
One of them is an action list, the other is a status list -- And yeah, annoyingly Phantom Flurry gives you a status called Phantom Flurry, and Apokalypsis gives you a status called Apokalypsis. It's not confusing at all, squeenix >:/

Reply:
Ah, I missed that one was a status list and the other an action list. Thanks SE!